src/Aspose.Imaging.Cloud.Sdk/Aspose.Imaging.Cloud.Sdk.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageId>Aspose.Imaging-Cloud</PackageId>
     <PackageVersion>19.7.0</PackageVersion>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageTags>aspose imaging cloud rest api sdk image bmp dicom dng djvu djv emf gif jpg jpe jpeg jpeg2000 jp2 jpx jpm j2k odg png psd tiff tif webp wmf cdr cmx pdf svg reverse search</PackageTags>
+    <PackageTags>aspose imaging cloud rest api sdk image bmp dicom dng djvu djv emf gif jpg jpe jpeg jpeg2000 jp2 jpx jpm j2k odg otg png psd tiff tif webp wmf cdr cmx pdf svg reverse search</PackageTags>
     <PackageProjectUrl>https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-dotnet</PackageProjectUrl>
```

Comment:
Please make package tags more descriptive in the next release. 

You may get some ideas from
https://www.nuget.org/packages/Aspose.Imaging/

Reply:
@aifeigin would you mind to include this topic for the next meeting? Thanks in advance.